sample/src/main/java/com/rocketchat/sample/RestApiRoomList.java

```diff
@@ -0,0 +1,174 @@
+package com.rocketchat.sample;
+
+import com.rocketchat.common.RocketChatException;
+import com.rocketchat.common.listener.ConnectListener;
+import com.rocketchat.common.listener.SimpleListCallback;
+import com.rocketchat.common.network.ReconnectionStrategy;
+import com.rocketchat.common.utils.Logger;
+import com.rocketchat.common.utils.Sort;
+import com.rocketchat.core.ChatRoom;
+import com.rocketchat.core.RocketChatClient;
+import com.rocketchat.core.callback.LoginCallback;
+import com.rocketchat.core.callback.RoomCallback;
+import com.rocketchat.core.factory.ChatRoomFactory;
+import com.rocketchat.core.model.Subscription;
+import com.rocketchat.core.model.Token;
+import com.rocketchat.core.model.attachment.Attachment;
+import com.rocketchat.core.provider.TokenProvider;
+
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * This class shows the lists fetched from a Room (e.g. its member list, file list, favorite message list and pinned message list).
+ *
+ * @author Filipe de Lima Brito (filipedelimabrito@gmail.com)
+ */
+public class RestApiRoomList {
```

Comment:
As discussed yesterday, just remove this file and provide just the snippet on `getRoomFilesByRoom()` method on the javadoc.

Reply:
Can we remove the entire sample module?

Reply:
@filipedelimabrito I would make a separate PR for that... Let's keep PRs focused.